task.sql

```diff
@@ -1,11 +1,7 @@
--- Use our database
 USE ShopDB; 
-
--- Some data should be created outside the transaction (here)
-
--- Start the transaction 
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, '2023-01-01');
 START TRANSACTION; 
+UPDATE Producs SET WarehouseAmount = WarehouseAmount - 1 WHERE ID = 1;
```

Comment:
Mistake in name of the table
```suggestion
UPDATE Products SET WarehouseAmount = WarehouseAmount - 1 WHERE ID = 1;
```